src/main/java/frc/robot/commands/TurnCmd.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DrivetrainSubsystem;
+
+public class TurnCmd extends CommandBase {
+  /** Creates a new TurnCmd. */
+  DrivetrainSubsystem drive;
+
+  boolean isdone;
+  double degree;
+
+  public TurnCmd(DrivetrainSubsystem drives, double degrees) {
```

Comment:
The `degrees` value is the number of degrees to turn to or the target heading?

i.e. does `TurnCmd(drives, 70)` turn the robot 70 more degrees or to the target heading of 70 degrees?